src/geocat/f2py/rcm2points_wrapper.py

```diff
@@ -22,67 +24,110 @@ def _rcm2points(lat2d, lon2d, fi, lat1d, lon1d, msg_py, opt, shape):
 
     fo = drcm2points(lat2d, lon2d, fi, lat1d, lon1d, xmsg=msg_fort, opt=opt)
     fo = np.asarray(fo)
-    fo = fo.reshape(shape)
+    fo = np.transpose(fo, axes=(1, 0))
 
     fort2py_msg(fi, msg_fort=msg_fort, msg_py=msg_py)
     fort2py_msg(fo, msg_fort=msg_fort, msg_py=msg_py)
 
     return fo
 
 
-# Outer Wrappers <funcname>()
-# These Wrappers are excecuted in the __main__ python process, and should be
+# Helper function that checks the validity of the input
+
+
+def _validity_check(lat2d, lon2d, fi, lat1d, lon1d):
```

Comment:
duplicated inner function name with dpres_plevel_wrapper.py both functions should be moved to, and imported from checks.py

Reply:
Is this an issue even though we don't import internal functions anywhere? (Though I agree that we could get checks.py work for these validity checks). Let me know if you agree with addressing this in a separate PR.

Reply:
separate PR works for me.